src/shared/browser-compatibility-utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Normalize a keyboard event key name.
+ *
+ * IE11 uses alternate key names for its keyboard events.
+ * If any abnormal keys are used (via IE11), this method
```

Comment:
Strictly speaking is is not just IE 11 that uses non-standard names for keys. There is an [MDN page](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) has details. Firefox <= 36 is the main browser that uses non-standard names for common keys, although I would imagine that the usage of Firefox versions that old is tiny to non-existent.

Reply:
I'll rework that doc so its correct